src/signal.ts

```diff
@@ -66,11 +64,7 @@ export async function generateQRCode(
   qrCodeOptions: QRCodeOptions = DEFAULT_QR_CODE_OPTIONS,
 ) {
   if (requestId === 'undefined') throw new Error(REQUEST_IS_MISSING_MESSAGE);
-  // TODO: Serialize data to standard URL for Deep-Links
-  qrCodeOptions.data = JSON.stringify({
-    requestId: requestId,
-    origin: url,
-  });
+  qrCodeOptions.data = `liquid://${url.replace('https://', '')}/?requestId=${requestId}`;
```

Comment:
shouldn't we strip `http` only from the prefix? I can us supporting url param values of http urls and that might mess things up